src/banman.cpp

```diff
@@ -69,78 +69,78 @@ void BanMan::ClearBanned()
         m_client_interface->BannedListChanged();
 }
 
-bool BanMan::IsBanned(CNetAddr netAddr)
+bool BanMan::IsBanned(CNetAddr net_addr)
 {
     LOCK(m_cs_banned);
     for (const auto& it : m_banned) {
-        CSubNet subNet = it.first;
-        CBanEntry banEntry = it.second;
+        CSubNet sub_net = it.first;
+        CBanEntry ban_entry = it.second;
 
-        if (subNet.Match(netAddr) && GetTime() < banEntry.nBanUntil) {
+        if (sub_net.Match(net_addr) && GetTime() < ban_entry.nBanUntil) {
             return true;
         }
     }
     return false;
 }
 
-bool BanMan::IsBanned(CSubNet subNet)
+bool BanMan::IsBanned(CSubNet sub_net)
 {
     LOCK(m_cs_banned);
-    banmap_t::iterator i = m_banned.find(subNet);
+    banmap_t::iterator i = m_banned.find(sub_net);
     if (i != m_banned.end()) {
-        CBanEntry banEntry = (*i).second;
-        if (GetTime() < banEntry.nBanUntil) {
+        CBanEntry ban_entry = (*i).second;
+        if (GetTime() < ban_entry.nBanUntil) {
             return true;
         }
     }
     return false;
 }
 
-void BanMan::Ban(const CNetAddr& netAddr, const BanReason& banReason, int64_t bantimeoffset, bool sinceUnixEpoch)
+void BanMan::Ban(const CNetAddr& net_addr, const BanReason& ban_reason, int64_t ban_time_offset, bool since_unix_epoch)
 {
-    CSubNet subNet(netAddr);
-    Ban(subNet, banReason, bantimeoffset, sinceUnixEpoch);
+    CSubNet sub_net(net_addr);
+    Ban(sub_net, ban_reason, ban_time_offset, since_unix_epoch);
 }
 
-void BanMan::Ban(const CSubNet& subNet, const BanReason& banReason, int64_t bantimeoffset, bool sinceUnixEpoch)
+void BanMan::Ban(const CSubNet& sub_net, const BanReason& ban_reason, int64_t ban_time_offset, bool since_unix_epoch)
 {
-    CBanEntry banEntry(GetTime());
-    banEntry.banReason = banReason;
-
-    int64_t normalized_bantimeoffset = bantimeoffset;
-    int64_t normalized_sinceUnixEpoch = sinceUnixEpoch;
-    if (bantimeoffset <= 0) {
-        normalized_bantimeoffset = m_default_ban_time;
-        normalized_sinceUnixEpoch = false;
+    CBanEntry ban_entry(GetTime());
+    ban_entry.banReason = ban_reason;
```

Comment:
Please don't use special-cases in scripted-diffs to negate part of a change, as that means that manual review is required.

Instead, please add a commit before the scripted-diff which renames any necessary conflicts.

Reply:
Do you mean rename `CBanEntry.banReason` to `CBanEntry.m_ban_reason` before the scripted diff?